packages/react-scripts/config/jest/babelTransform.js

```diff
@@ -1,4 +1,3 @@
-// @remove-file-on-eject
 /**
```

Comment:
You will need to wrap the copyright header in a `remove-on-eject` block, like the rest of the project does on files that are written to the user's project after ejecting.

Reply:
Good catch. Fixed in https://github.com/facebook/create-react-app/pull/10091/commits/d110fac0f06c80849e316476bd1409a006f899d3.